releasenotes/notes/qdrift-as-product-formula-044a37a106a45a47.yaml

```
@@ -0,0 +1,6 @@
+---
+prelude: >
+    Reformulate the former QDrift class in qiskit.opflow.evolutions.trotterizations as a ProductFormula instead of a TrotterizationBase.
+other:
```

I think we can call this a new feature 😄 Could add a small example code on how to call it? You can write it as
```
.. code-block:: python

    example here
```
```suggestion
features:
```

---

added :)